Makefile

```diff
@@ -123,7 +123,7 @@ test-doc-clean:
 	make doc-clean DOC=pb
 	make doc-clean DOC=presentation
 
-test: test-clean test-pc-emul test-sim test-fpga test-asic test-doc
+test: test-clean test-pc-emul test-sim test-fpga test-doc
```

Comment:
any test target for asic ?

Reply:
There is one, but it is not fully functional.